packages/qdrant-loader/src/qdrant_loader/core/worker/handlers.py

```
@@ -0,0 +1,198 @@
+# ---------------------------------------------------------------------------
+
+
+class IngestionJobHandler(BaseJobHandler):
+    """Concrete handler for BULK_INGEST and INCREMENTAL_PULL jobs.
+
+    Delegates to PipelineOrchestrator for document processing.
+    """
+
+    def __init__(
+        self,
+        orchestrator: Any,
```

Ruff (0.15.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `orchestrator`

(ANN401)